src/main/java/com/springboot/star_wars_character_popularity/app/controller/CharacterController.java

```diff
@@ -47,4 +51,28 @@ public CharacterController(CharacterService characterService){
 
         return new ResponseEntity<String>("Character deleted successfully !", HttpStatus.OK);
     }
+
+    //------------ Thymeleaf controllers ----------------
+
+    /*
+    @GetMapping("/hello/hello")
+    public String sayHello(Model theModel){
+        theModel.addAttribute("theDate", new java.util.Date());
+
+        return "helloworld";
+    }
+     */
+
+    @GetMapping("/hello/hello")
```

Comment:
I think you should reduce this to @GetMapping("/hello") or even move this to a new Controller to test it out.